rdr_service/dao/genomics_dao.py

```diff
@@ -1623,3 +1623,19 @@ def truncate_value(value, max_length):
             value = value[:max_length]
 
         return is_truncated, value
+
+
+class GenomicCloudRequestsDao(UpdatableDao):
+
+    validate_version_match = False
```

Comment:
I'm not clear on how/where this is used?  (although I see it defined in other classes as well)